src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  flex-shrink: 0;
+  width: get-calendar-width();
+
+  &__day {
+    @include flex-center;
+    @include square($square-size);
+    background-color: #eee;
+    border: $border-width solid #000;
+    transition: transform 0.5s, background-color 0.5s;
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(2n + #{$i}) {
+        &::before {
+          display: block;
+          content: "#{$i}";
+        }
+      }
+    }
+  }
+
+  @each $day, $number-of-day in $days {
+    &--start-day_#{$day} {
+      .calendar__day:first-child {
+        margin-left: get-margin-size($number-of-day);
+      }
+    }
+  }
```

I think {}  are not needed here, you can remoe them like:
```suggestion
  @each $day, $number-of-day in $days {
    &--start-day_#{$day} .calendar__day:first-child {
        margin-left: get-margin-size($number-of-day);
      }
  }
```